### PR TITLE
Structural/membrane damping matrix

### PR DESCRIPTION
**Pull request by AndreasWinterstein**
This pull request fixes the calculation of the body force and the damping matrix in the membrane element. Additionally it adds patch tests for static and dynamic analysis for 3D3N and 3D4N elements.

The linear solver type in several tests had to be changed in order to make them run with the new "linear_solvers_speed" variable in the structural mechanics solver. The tests for sensitivities are skipped since they are currently failing.

The membrane element has also been tested for larger structural mechanics problems. It also works fine there.

**In any case, a cleanup of the code for the membrane element is necessary. This will be part of another pull request.**

### CODE REVIEW

**Review by philbucher — COMMENTED** [review]
Please wait with merging, I would like to discuss some things in person later today

**Comment by loumalouomega** [comment]
I restarted the Travis, I will give my review, but I assume you will discuss in detail with @philbucher

File: applications/StructuralMechanicsApplication/custom_elements/prestress_membrane_element.cpp [hunk 1]
```diff
@@ -175,6 +175,60 @@ void PrestressMembraneElement::CalculateRightHandSide(
...
+    const SizeType num_dofs = number_of_nodes * 3;
+
+    if ( rDampingMatrix.size1() != num_dofs )
+        rDampingMatrix.resize( num_dofs, num_dofs, false );
+
+    noalias( rDampingMatrix ) = ZeroMatrix( num_dofs, num_dofs );
+
+    // 1.-Calculate StiffnessMatrix:
+
+    MatrixType StiffnessMatrix  = Matrix();
```
Comment (loumalouomega) [inline applications/StructuralMechanicsApplication/custom_elements/prestress_membrane_element.cpp]: For the variables declared in a method is snake case, no canmel case

**Comment by philbucher** [comment]
@loumalouomega I'd wati with reviewing, some things will change

File: applications/StructuralMechanicsApplication/custom_elements/prestress_membrane_element.cpp [hunk 2]
```diff
@@ -175,6 +175,60 @@ void PrestressMembraneElement::CalculateRightHandSide(
...
+    VectorType ResidualVector  = Vector();
+
+    CalculateAll(StiffnessMatrix, ResidualVector, rCurrentProcessInfo, true, false);
+
+    // 2.-Calculate MassMatrix:
+
+    MatrixType MassMatrix  = Matrix();
```
Comment (loumalouomega) [inline applications/StructuralMechanicsApplication/custom_elements/prestress_membrane_element.cpp]: Idem

**Review by loumalouomega — DISMISSED** [review]
Minor comments

**Review by loumalouomega — DISMISSED** [review]
OK

**Review by loumalouomega — DISMISSED** [review]

**Comment by philbucher** [comment]
@AndreasWinterstein the test of the 4-Node element takes 0.2 seconds, which strictly spoken is too long for a small-suite test
We should speak tmr

**Review by philbucher — APPROVED** [review]
tests will be refactored in the next PR

**Pull request merged**